source/events/edit.html.md

```diff
@@ -5,7 +5,7 @@ title: Edit RDO events
 # Editing the RDO events calendar
 
 The RDO events calendar is maintained as part of the
-[community.redhat.com](community.redhat.com) events site. To edit or add
+[community.redhat.com](https://www.community.redhat.com) events site. To edit or add
```

Comment:
The url www.community.redhat.com is not valid.

Reply:
interesting... looks like http://community.redhat.com/ is just fine but www isn't. Is that something that should be fixed in the DNS side of things? Or are we not worried about it?